.github/workflows/agw.branch_push.yml

```diff
@@ -24,11 +25,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Install tox
-        run: pip install tox
+        run: pip install tox==3.21.4
       - name: Run tests using tox
         run: |
           sudo apt update
-          sudo apt install libsystemd-dev
+          sudo apt install libsystemd-dev=245.4-4ubuntu3.19
```

Comment:
I am not sure we want to pin this version, as we are already pinning the version of Ubuntu to an LTS release, so it should only get bug fixes and security fixes.

Reply:
Same for all workflows changed in this PR

Reply:
Agreed.